tests/safeds/data/tabular/containers/_table/test_get_similar_columns.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+from safeds.data.tabular.containers import Table
+from safeds.exceptions._data import UnknownColumnNameError
+
+
+@pytest.mark.parametrize(
+    ("table", "column_name", "expected"),
+    [
+        (Table({"column1": ["col1_1"], "col2": ["col2_1"], "cilumn2": ["cil2_1"]}), "col1", ["column1", "col2"]),
+        (Table({"column1": ["col1_1"], "col2": ["col2_1"], "cilumn2": ["cil2_1"]}), "clumn1", ["column1", "cilumn2"]),
```

Comment:
May also want to include testcases where some of the column names are way off, so it's easier to see what columns are the "right" ones to suggest.

Reply:
done